examples/platforms/da15000/da15000.ld

```diff
@@ -0,0 +1,230 @@
+/*
```

Comment:
Not straight forward operation as it is related to chip architecture. Some basic cleanup is in place. Further simplification will be handled later on.